hedy.py

```diff
@@ -33,6 +35,12 @@
 # Python keywords need hashing when used as var names
 reserved_words = ['and', 'except', 'lambda', 'with', 'as', 'finally', 'nonlocal', 'while', 'assert', 'False', 'None', 'yield', 'break', 'for', 'not', 'class', 'from', 'or', 'continue', 'global', 'pass', 'def', 'if', 'raise', 'del', 'import', 'return', 'elif', 'in', 'True', 'else', 'is', 'try']
 
+# Let's retrieve all keywords dynamically from the cached KEYWORDS dictionary
+indent_keywords = []
+for lang, keywords in KEYWORDS.items():
+    for keyword in ['if', 'for', 'repeat']:
```

Comment:
We might miss `while` here (I know we did not have it in the original but I think that was a mistake!)

Reply:
Whoops! Will add.

Reply:
So that was my bad, originally :)